src/Core/Components/Dialog/Services/DialogInstance.cs

```diff
@@ -20,6 +23,8 @@ public DialogInstance(Type? type, DialogParameters parameters, object content)
 
     public object Content { get; internal set; } = default!;
 
+    public IJSObjectReference PreviouslyFocusedElement { get; }
```

Comment:
Can you change this property to `internal`?

Reply:
Done!